pt/lessons/specifics/nerves.md

```diff
@@ -0,0 +1,262 @@
+
+De qualquer forma, nós sugerimos usar um Raspberry Pi, já que ele tem um LED controlável onboard. Também é recomendável ter uma tela conectada ao seu dispositivo alvo já que isso vai simplificar o debug usando IEx.
+However, we would suggest using a Raspberry Pi, due to it having controllable LED onboard. It is also advisable to have a screen connected to your target device as this will simplify debugging using IEx.
+
+## Configuração
+
+O projeto Nerves tem um excelente [Guia de introdução](https://hexdocs.pm/nerves/getting-started.html), mas a quantidade de detalhes lá pode ser assustadora para alguns usuários. Em vez disso, esse tutorial vai tentar apresentar "menos palavras, mais código".
+
+Primeiramente, você vai precisar de um ambiente configurado. Você pode encontrar o guia na parte de [Instalação](https://hexdocs.pm/nerves/installation.html) da wiki do Nerves. Por favor, tenha certeza de que você tem a mesma versão tanto do OTP quanto do Elixir mencionadas no guia. Não utilizar a versão correta pode causar problemas conforme você progride. No momento em que esse guia foi escrito, qualquer versão de Elixir (compilada com Erlang/OTP 21) deve funcionar.
+
+Depois de configurar, você deve conseguir construir seu primeiro projeto Nerves!
+
+Nosso objetivo será chegar ao "Hello world" do desenvolvimento embarcado: um LED piscando controlado por uma API HTTP simples.
+
+## Criando um projeto
+
+Para gerar um novo projeto, rode `mix nerves.new network_led` e responda `Y` à pergunta sobre obter e instalar as dependências.
+
+Você deve ver a seguinte saída:
+
+```
+Your Nerves project was created successfully.
+
+You should now pick a target. See https://hexdocs.pm/nerves/targets.html#content
+for supported targets. If your target is on the list, set `MIX_TARGET`
+to its tag name:
+
+For example, for the Raspberry Pi 3 you can either
+  $ export MIX_TARGET=rpi3
+Or prefix `mix` commands like the following:
+  $ MIX_TARGET=rpi3 mix firmware
+
+If you will be using a custom system, update the `mix.exs`
+dependencies to point to desired system's package.
+
+Now download the dependencies and build a firmware archive:
+  $ cd network_led
+  $ mix deps.get
+  $ mix firmware
+
+If your target boots up using an SDCard (like the Raspberry Pi 3),
+then insert an SDCard into a reader on your computer and run:
+  $ mix firmware.burn
+
+Plug the SDCard into the target and power it up. See target documentation
+above for more information and other targets.
+```
+
+Nosso projeto foi gerado e está pronto para ser transferido para nosso dispositivo de teste! Vamos tentar agora!
+
+No caso de um Raspberry Pi 3, você define `MIX_TARGET=rpi3`, mas você pode mudar isso para se adequar ao hardware que você tem dependendo do hardware alvo (veja a lista na [documentação do Nerves](https://hexdocs.pm/nerves/targets.html#content)).
```

Comment:
what do you think instead use:
_No caso de um Raspberry Pi 3, você define `MIX_TARGET=rpi3`,..._

use something like these:
_No nosso exemplo iremos utilizar um Raspberry Pi3, então utilizaremos a variavel de ambiente `MIX_TARGET=rpi3`, correspondente ao hardware utilizado,..._

Reply:
I don't think this change keeps the same meaning to the translation.  This kind of writing structure is being used on multiple places, and refers to the fact that multiple hardwares are being cited in the text for different reasons. I honestly don't think this is worth changing